### PR TITLE
feat: log version and settings on startup

### PR DESCRIPTION
**Pull request by igorbernstein2**
Add some informative log messages about the client on startup. A similar feature existed in 1x, but got lost in the upgrade 2.0. Will print the artifact name, version and settings for each new connection

### CODE REVIEW

File: moo/pom.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>bigtable-client-parent</artifactId>
+    <groupId>com.google.cloud.bigtable</groupId>
+    <version>2.1.1</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>moo</artifactId>
```
Comment (mutianf) [inline moo/pom.xml:12]: moo? lol

**Review by mutianf — COMMENTED** [review]
Please fix the pom file :P

File: moo/pom.xml (hunk 1 above)
Comment (igorbernstein2) [reply moo/pom.xml]: 😳

**Review by mutianf — APPROVED** [review]
lgtm

**Pull request merged**